amxmodx/meta_api.cpp

```diff
@@ -135,9 +135,14 @@ cvar_t init_amxmodx_debug = {"amx_debug", "1", FCVAR_SPONLY};
 cvar_t init_amxmodx_mldebug = {"amx_mldebug", "", FCVAR_SPONLY};
 cvar_t init_amxmodx_language = {"amx_language", "en", FCVAR_SERVER};
 cvar_t init_amxmodx_cl_langs = {"amx_client_languages", "1", FCVAR_SERVER};
+cvar_t init_amxmodx_perflog = { "amx_perflog", "1.0", FCVAR_SPONLY };
```

Comment:
`amx_preflog_ms` would be more descriptive, imo.

Reply:
@ShootingKing-AM if we end up with microseconds then it should be `amx_perflog_us`.

Reply:
Yes I can do it

Reply:
@WPMGPRoSToTeMa but this is milliseconds, with possible to use with microseconds precision.
I think amx_preflog_ms better than us?

Reply:
@UnrealKaraulov ah, it's float, I see. Yes, then of course we should name it `amx_perflog_ms`.